src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -53,44 +68,58 @@
     }
 
     @GetMapping("/child")
+    @ApiOperation(value = "Find a registered requested user's children data")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK"),
+            @ApiResponse(code = 404, message = "NOT FOUND")})
     public ResponseEntity<List<ChildDto>> getChildren(Principal principal) {
-        return new ResponseEntity<>(userService.getChildren(getUser(principal.getName())), HttpStatus.OK);
+        List<ChildDto> children = userService.getChildren(getUser(principal.getName()));
+        return children.isEmpty() ?
+                new ResponseEntity<>(HttpStatus.NOT_FOUND) :
+                new ResponseEntity<>(children, HttpStatus.OK);
     }
 
     @PostMapping("/child")
-    public ResponseEntity<HttpStatus> addChildren(@RequestBody
-                                                  @Size(min = 1, message = "{exception.invalid.dto}")
-                                                          List<@Valid ChildDto> childrenDto,
-                                                  Principal principal) {
+    @ApiOperation(value = "Add children data for a registered requested user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK"),
+            @ApiResponse(code = 400, message = "BAD REQUEST")})
+    @ResponseStatus(HttpStatus.OK)
+    public void addChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}")
```

Comment:
It would be nice to return saved children but it requires additional changes on the service layer. So I prefer leaving it as is and don't change it in scope of current ticket.